src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
```

Comment:
I think we should just have a mount in `NewCreateServiceModal` that provides a way for additional options to be injected but the injection (component registration against a mount point) should be handled in a plugin and this open source code shouldn't need to know anything about `hasCapability`.